packages/nc-gui/composables/useUIPermission/rolePermissions.ts

```diff
@@ -1,51 +1,65 @@
 import { ProjectRole, Role } from '~/lib'
 
-const rolePermissions = {
+const rolePermissions: Record<Role | ProjectRole, '*' | Partial<Record<'include' | 'exclude', Record<string, boolean>>>> = {
```

Comment:
Please remove this typing.
`rolePermissions` is typed as a constant (hence the `as const` cast at the end)

That means the object itself is the interface of the object and adding an additional type as you did just ruins the intention of `as const`